student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name: 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Since you are using the colon (:) notation for your parameters, you need use them correctly for your Parent class assignments or it will throw errors.

  ```
  super(age, name: name, parent_permission: parent_permission)
  ```

  ![ws](https://user-images.githubusercontent.com/36542986/199437981-c3b5d099-2cc1-45b8-b423-27ee2caade60.jpg)